### ISSUE TITLE
Dialogs: Resize selection list automatically to content with `Get Selection From User`

### ISSUE THREAD

**Issue by MisterChild** [robotframework/robotframework#2517]
Enhancement:

With having long list elements, the dialog window won't resize width to longest element.

Adding the line `self._listbox.config(width=0)` to function "_create_selector()" in dialogs_py.py will fix this.

```
class SelectionDialog(_TkDialog):

    def __init__(self, message, values):
        _TkDialog.__init__(self, message, values)

    def _create_selector(self, parent, values):
        self._listbox = Listbox(parent)
        for item in values:
            self._listbox.insert(END, item)
        # resize window width
        self._listbox.config(width=0)
        return self._listbox
```

Only tested on Ubuntu 14.04 with RFW 2.9.2.

**Comment by pekkaklarck** [comment]
Could you create a pull request with a fix and preferably with a new or updated test to demonstrate the problem and validate the fix?

**Comment by pekkaklarck** [comment]
@MisterChild, do you have interested to create a pull request with the fix and a test? Shouldn't be too hard and I can help if needed.

**Comment by MisterChild** [comment]
Hi, I'll go on a long journey from next week until may. If I find time next days I try to create a pull request before the trip. But I think may is more realistic. ☺

**Comment by pekkaklarck** [comment]
Getting this done ought to be easy:

- You already have implementation ready.
- No need to update documentation.
- This can be tested by adding some longer items into [an existing tests](https://github.com/robotframework/robotframework/blob/master/atest/testdata/standard_libraries/dialogs/dialogs.robot#L51)

If you are new to pull requests and contributing in general, start by reading `CONTRIBUTING.rst` in the project root.

The acceptance test system is documented in `atest/README.rst`, but in this case you basically can just execute the test case file referenced above directly:

    ./rundevel.py atest/testdata/standard_libraries/dialogs/dialogs.robot

**Comment by pekkaklarck** [comment]
PR #2517 looks promising.

**Comment by pekkaklarck** [comment]
This was implemented by PR #2533. I tested the fix also on Windows with Python and with PyPy on Linux. Also verified that with Jython and IronPython the dialog is resized correctly without fixes.

Thanks @MisterChild for a PR and safe journeys!
